src/components/user/IFXUserList.vue

```diff
@@ -65,13 +75,23 @@ export default {
     <IFXPageHeader>
       <template #title>{{listTitle}}</template>
       <template #actions>
-        <IFXSearchField :search.sync='search'/>
-        <v-checkbox class="action-item" label="Include disabled" v-model="includeDisabled"></v-checkbox>
-        <IFXActionSelect
-          class='action-item'
-          :selectedItems.sync='selected'
-          :actionKeys="['activateUserLogin', 'deactivateUserLogin', 'addMailingTo', 'addMailingCC', 'addMailingBCC']"
```

Comment:
Can users be activated and deactivated in some other way now?

Reply:
Will replace shortly